From c02f30b33e5bd0ef600708241702b2fad3de3ff4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 16 Aug 2017 18:58:20 -0700
Subject: [PATCH] winsys/svga/drm: Include sys/types.h

vmw_screen.h uses dev_t which is defines in sys/types.h
this header is required to be included for getting dev_t
definition. This issue happens on musl C library, it is hidden
on glibc since sys/types.h is included through another
system headers

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/gallium/winsys/svga/drm/vmw_screen.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/winsys/svga/drm/vmw_screen.h b/src/gallium/winsys/svga/drm/vmw_screen.h
index f21cabb..4c972fd 100644
--- a/src/gallium/winsys/svga/drm/vmw_screen.h
+++ b/src/gallium/winsys/svga/drm/vmw_screen.h
@@ -41,6 +41,7 @@
 #include "svga_winsys.h"
 #include "pipebuffer/pb_buffer_fenced.h"
 #include <os/os_thread.h>
+#include <sys/types.h>
 
 #define VMW_GMR_POOL_SIZE (16*1024*1024)
 #define VMW_QUERY_POOL_SIZE (8192)
